ansible/config_sonic_basedon_testbed.yml

```diff
@@ -771,6 +771,13 @@
 
         when: "'dualtor-mixed' in topo or 'dualtor-aa' in topo"
 
+      - name: Apply device conn to config db
+        apply_device_conn_sonic:
```

Comment:
why do we need this for applying sonic config? 
much of the config DB config will happen via #13990

Reply:
Are you sure the AutoNeg value in minigraph will be applied into the config DB?